src/helpers/ingressHandler.ts

```diff
@@ -0,0 +1,31 @@
+import { InvocationContext } from '@azure/functions';
+import { CosmosDbHelper } from './cosmosDbHelper';
+import { OpenAiHelper } from './openAiHelper';
+import { SmsEgressHandler } from './smsEgressHandler';
+
+export async function ingressHandler(message: string, context: InvocationContext): Promise<void> {
+
+    const userID = (context.triggerMetadata.userProperties as UserProperties).address;
```

Comment:
I like userId more :P - if we stick with userID throughout the code, I will let this one slip :laughing:  :+1: